src/main/java/org/betonquest/betonquest/api/CountingObjective.java

```diff
@@ -48,9 +49,29 @@ public CountingObjective(final Instruction instruction, final String notifyMessa
         defaultNotifyMessageName = notifyMessageName;
     }
 
+    /**
+     * This checks the {@link VariableNumber} absence to a variable and ensures the absolute value is positive
+     * if no variable is present
+     *
+     * @param variableNumber The variable to check
+     * @throws InstructionParseException if the stored value is explicit negative
+     */
+    protected final void preCheckAmountNotLessThanOne(final VariableNumber variableNumber)
+            throws InstructionParseException {
+        if (variableNumber.explicitLessThanOne()) {
+            throw new InstructionParseException("Amount cannot be less than 1");
+        }
+    }
+
     @Override
     public final String getDefaultDataInstruction() {
-        return Integer.toString(targetAmount);
+        return targetAmount.toString();
+    }
+
+    @Override
+    public String getDefaultDataInstruction(final Profile profile) {
+        final int value = targetAmount.getInt(profile);
+        return value > 0 ? String.valueOf(value) : "1";
```

Comment:
this have to be either moved to the sub classes or an boolean for >0 has to be introduced

Reply:
This method is overriden in the BlockObjective